pkg/cluster/managed.go

```diff
@@ -168,3 +172,18 @@ func (c *Cluster) deleteNodePasswdSecret(ctx context.Context) {
 		logrus.Warnf("failed to delete old node password secret: %v", err)
 	}
 }
+
+// handlerNoEtcd wraps a handler with an error message indicating that etcd is not deployed.
+func handlerNoEtcd(handler http.Handler) http.Handler {
+	r := mux.NewRouter().SkipClean(true)
+
+	// Wildcard route for anything after /db/
+    r.HandleFunc("/db/{_:.*}", func(resp http.ResponseWriter, r *http.Request) { 
+        util.SendError(errors.New("etcd datastore disabled"), resp, r, http.StatusBadRequest)
+    })
```

Comment:
inconsistent indentation
```suggestion
	// Wildcard route for anything after /db/
	r.HandleFunc("/db/{_:.*}", func(resp http.ResponseWriter, r *http.Request) { 
		util.SendError(errors.New("etcd datastore disabled"), resp, r, http.StatusBadRequest)
	})
```